Add Countries

MustaphaTR opened the pull request:
This differs a lot from original. Because Countries in original game was a buggy mess.

- England's Armor and France's ROF bonus was buggy in original, making them weak/fire slower. They properly get their bonuses here.
- Russia's cost bonus has a wierd calculation error. Making stuff above $300 original cost $1-4 cheaper then actual 10%. I did NOT fix that. Because i think wierd costs of Russia gives a classic feeling and is too low to do a real difference. But this cost bonus was also effecting build time in original and was most likely a bug, becuase there is [a seperate ini entry for BuildTime](https://github.com/OpenRA/raclassic/blob/master/ref/Rules.ini#L313), not sure if works or not, no vanilla faction uses it.
- Ukraine's speed bonus in original only effected ground units, it effects air units too here.
- I added Spain, Greece and Turkey, which we don't have main OpenRA RA mod.
- In original Spain, Greece and Turkey had no bonus, i gave some bonuses to them here.
- Turkey gets 10% build time bonus.
- Greece gets 10% sight bonus.
- Spain gets 10% range bonus.
- I kept original Allied and Soviet countries, they have no bonus. And "Any" does not select them, so you won't be in a disadvantage by getting them, unless you want to.

Tho, i'm not sure about bonuses of Spain and Greece, or tbh, if we want that 3 countries at all. 10% is too low for a range bonus. Greece is allied country, they have GPS and FoW is disabled by default so sight bonus don't do much, and is literally useless in late game. I think we can make them better by changing them to +1 rather than 10%, and/or replace sight bonus with 10% more power output from Power Plants, could be more useful, but want to hear other's opinions too.

cjshmyr commented:
I'm ok with adding the unused countries, in favor of the +1 over 10% for the sight/range. Also like the power idea if sight's no good.

MustaphaTR commented:
Updated, changed Spain's bonus to +1 range, Greece's +10% Power. I used RangeMultiplier on some units, but sepeate armaments on some, ones who has sepeate armaments are units with more than one weapon with different ranges, as it is not possible to make RangeMultiplier effect a specific armament.
